cvmfs/shortstring.h

```diff
@@ -87,6 +87,15 @@ class ShortString {
     this->length_ = new_length;
   }
 
+  void Truncate(unsigned char new_length) {
```

Comment:
```suggestion
  void Truncate(unsigned new_length) {
```

It seems unnecessary to restrict `new_length` to 255 characters.